test/built-ins/Number/prototype/toFixed/length.js

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2017 Michael "Z" Goddard. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+es6id: 20.1.3.3
```

Comment:
while it does not hurt leaving es6id, the current appropriate tag is esid as in this example: https://github.com/tc39/test262/pull/824/files#diff-971804c3f7e390b76d945b7285de923aR5